### ISSUE TITLE
Feature request: Pause inbox

### ISSUE THREAD

**Issue by orschiro** [nylas/N1#2022]
Dear all, 

Sometimes I wish there was a way to pause my inbox from receiving new emails, yet being able to view existing emails. 

Does that resonate with others?

Thankfully,

~Robert

**Comment by zlatkobijelic** [comment]
Yeah I think this would a great feature to implement. Would you say that emails would still populate but you just dont receive a **notification**? Or when you pause, it doesn't populate at all until you hit _resume_ and then all emails flood the inbox?

**Comment by rakeshdas1** [comment]
@zlatkobijelic I think the second option sounds the best. That way, I can deal with email in groups and not get distracted by new ones coming in.

**Comment by orschiro** [comment]
Dear @zlatkobijelic and @rakeshdas1,

I am very appreciative for your responses.

>  Or when you pause, it doesn't populate at all until you hit resume and then all emails flood the inbox?

Precisely this one. Not only muting notifications but actually not fetching any new emails from the server!

Warmly,

~Robert

**Comment by molstrangler** [comment]
I would have thought N1 should recognise when the user has use the OS option rather than making the user do two actions for this.

For Windows 10 there is a 'Quiet Hours' option and in OS in the Notification Centre there is 'Do not disturb'.

**Comment by orschiro** [comment]
@molstrangler, 

good point! However, not all OS support this. For instance, on Ubuntu 16.04 I am not aware of an equivalent to "Quiet Hours".

~Robert

**Comment by molstrangler** [comment]
humm!!!

Maybe N1 could have the built-in option for all builds but the Windows & OS X builds also take notice of the OS function and flick the switch in N1 when the user uses the OS option.

**Comment by orschiro** [comment]
> Maybe N1 could have the built-in option for all builds but the Windows & OS X builds also take notice of the OS function and flick the switch in N1 when the user uses the OS option.

This sounds like a great proposal!

**Comment by bengotow** [comment]
Hey folks!

It sounds like there are two separate concepts here:

1. Integration with "Do Not Disturb / Quiet Hours": The way these features work, you would still receive new mail (nothing is really paused), but you wouldn't get notifications about incoming mail. This is tracked in a separate issue here: https://github.com/nylas/N1/issues/715.

2. Ability to totally pause all incoming mail. Not just the notifications / sounds, but the actual appearance of the mail. This isn't currently on the roadmap, but could be implemented as a switch per account. As a side note, you could currently achieve this by blocking N1's access to api.nylas.com.

**Comment by molstrangler** [comment]
I would vote to follow the the specific OSes guideline.  One way to really annoy a user would be for N1 to work differently to other apps on their chosen OS, to follow the OS guide lines for 'do not disturb'.

We know OS X & Windows 10 support the OS level 'do not disturb' option and I would think the users of these different OSes would expect consistency across all their apps for their OS.

I know this could mean N1 may work differently between OSes but I think it's important to follow how the OS works.

I cannot speak for Linux!

**Comment by orschiro** [comment]
FYI: Quick and dirty workaround script for Linux users to temporarily mute notifications: 

https://gist.github.com/orschiro/46d75fce983ebf4f5d9aa199cd781606

**Comment by levibostian** [comment]
I currently use [Inbox pause](http://inboxpause.com/) for GMail and it is the reason why I am still using web based clients. This is would I would love in N1. Every day at noon I unpause my inbox and then repause it which then dumps all of my paused emails into my inbox so I can quickly run through them all at once. 

So the value this feature would be providing is to allow me to check my email at a designated time each day and hide everything else.

**Comment by ismith** [comment]
The thread above seems to be talking about a global setting; I'd like to be able to do this per-account.  (So, e.g., "I'm on vacation" -> "no work email unless I specifically request it".)
